### PR TITLE
Augmentations

### PR DESCRIPTION
**Pull request by ewgoedeke**
separate PR for the Augmentation functionality

### CODE REVIEW

File: gluefactory/settings.py
Comment (RaphaelKreft) [inline gluefactory/settings.py:1]: Don't change the settings.py file!

File: gluefactory/datasets/oxford_paris_mini_1view_jpldd.py [hunk 1]
```diff
@@ -162,6 +177,7 @@ def __init__(self, conf, image_sub_paths: list[str], split):
 
         self.img_dir = DATA_PATH / conf.data_dir
         self.dlsd_kp_gt_folder = self.img_dir.parent / "deeplsd_kp_gt"
+        self.dlsd_line_gt_folder = self.img_dir.parent / "deeplsd_line_gt"
```
Comment (RaphaelKreft) [inline gluefactory/datasets/oxford_paris_mini_1view_jpldd.py:180]: What has this line to do with Augmentations? I think it was added accidentally?

File: .gitignore
Comment (RaphaelKreft) [inline .gitignore:1]: you should only add general important items to the gitignore for all. ie. please remove the image_list entry

**Review by RaphaelKreft — CHANGES_REQUESTED** [review]
Please also add augmentations to minidepth, so we have it there as well.
Otherwise after addressing the nitpicks, this PR looks fine

File: gluefactory/settings.py
Comment (ewgoedeke) [reply gluefactory/settings.py:1]: its reverted now

File: gluefactory/datasets/oxford_paris_mini_1view_jpldd.py (hunk 1 above)
Comment (ewgoedeke) [reply gluefactory/datasets/oxford_paris_mini_1view_jpldd.py]: yes this is unrelated to augmentations and is intended for the binary heatmaps

**Review by RaphaelKreft — APPROVED** [review]
lgtm now

**Pull request merged**